crates/bevy_core_pipeline/src/core_3d/mod.rs

```diff
@@ -78,7 +81,8 @@ impl Plugin for Core3dPlugin {
                     sort_phase_system::<Opaque3d>.in_set(RenderSet::PhaseSort),
                     sort_phase_system::<AlphaMask3d>.in_set(RenderSet::PhaseSort),
                     sort_phase_system::<Transparent3d>.in_set(RenderSet::PhaseSort),
-                ),
+                )
+                    .ignore_stepping(),
```

Comment:
I don't really understand why `rustfmt` is indenting this, but it happens everywhere I added `ignore_stepping()` to a tuple.